addons/binding/org.openhab.binding.mihome/ESH-INF/thing/channel.xml

```diff
@@ -119,6 +119,17 @@
         <state min="0" max="100" step="0.1" pattern="%.1f %%" readOnly="true">
         </state>
     </channel-type>
+    
+    <channel-type id="pressure">
+        <item-type>Number</item-type>
+        <label>Pressure</label>
+        <category>Pressure</category>
+        <tags>
+            <tag>weather</tag>
```

Comment:
@kaikreuzer if that's the case then [this ](https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#default-tags)link needs updating as that's where I got it from.